pkg/apimachinery/patch/patch_test.go

```diff
@@ -0,0 +1,78 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2024 Red Hat, Inc.
+ *
+ */
+
+package patch_test
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"kubevirt.io/kubevirt/pkg/apimachinery/patch"
+)
+
+var _ = Describe("PatchSet", func() {
+
+	It("Patch remove operation", func() {
```

Comment:
nit: can we improve the test flow descriptions?
Right now this will result in:
"Patchset [It] Patch remove operation".
I think that we can achieve something like:
"PatchSet [It] should generate correct patch remove operation"

Reply:
Done